backend/app/controllers/books_controller.rb

```diff
@@ -4,9 +4,12 @@
 class BooksController < ApplicationController
   before_action :authorize_request
 
+  def user_books_index
+    render(json: BookSerializer.new(user_books, pagination(user_books)), status: :ok)
+  end
```

Comment:
`nit` 

Another way to do this is by creating an entirely new controller with this endpoint:

`/my_books`